dev/TeachingTip/TeachingTip.cpp

```diff
@@ -867,6 +952,18 @@ void TeachingTip::OnActionButtonClicked(const winrt::IInspectable&, const winrt:
 void TeachingTip::OnPopupOpened(const winrt::IInspectable&, const winrt::IInspectable&)
 {
     StartExpandToOpen();
+
+    if (auto peer = winrt::FrameworkElementAutomationPeer::FromElement(*this))
+    {
+        if (auto teachingTipPeer = peer.as<winrt::TeachingTipAutomationPeer>())
+        {
+            auto notificationString = StringUtil::FormatString(
+                ResourceAccessor::GetLocalizedStringResource(SR_TeachingTipNotification),
+                winrt::ApplicationModel::Package::Current().DisplayName(),
```

Comment:
```suggestion
                winrt::ApplicationModel::Package::Current().DisplayName().data(),
```

Reply:
I also just realized you're using Package.Current which won't work in an unpackaged app, i don't think. Shouldn't we just use the TeachingTip's AutomationName? Or maybe fall back to that if Package.Current is null?

Reply:
Oh i see the second param is the popup's name. Hmm. I dunno. Maybe grab the top level window's name?

Reply:
Hmm, I should probably have a fallback for when the application name isn't available regardless.